frontends/web/src/components/layout/header.module.css

```diff
@@ -70,6 +75,7 @@
 .sidebarToggler {
     width: 32px;
     height: 32px;
+    margin-bottom: var(--space-quarter);
     margin-right: var(--spacing-default);
```

Comment:
I was a bit confused that the space between the hamburger and the title on medium screen was larger than small screen. But this is already on master, so lets keep as is.